Makefile

```diff
@@ -161,39 +157,12 @@ fmt:
 	@$(GOFMT) -e -s -l -w $(ALL_SRC)
 	./scripts/updateLicenses.sh
 
-.PHONY: lint-gosec
-lint-gosec:
-	time gosec -quiet -exclude=G104,G107 ./...
-
-.PHONY: lint-staticcheck
-lint-staticcheck:
-	@cat /dev/null > $(LINT_LOG)
-	time staticcheck ./... \
-		| grep -v \
-			-e model/model.pb.go \
-			-e proto-gen \
-			-e _test.pb.go \
-			-e thrift-gen/ \
-			-e swagger-gen/ \
-		>> $(LINT_LOG) || true
-	@[ ! -s "$(LINT_LOG)" ] || (echo "Detected staticcheck failures:" | cat - $(LINT_LOG) && false)
-
 .PHONY: lint
-lint: lint-staticcheck lint-gosec
-	$(GOVET) ./...
-	$(MAKE) go-lint
-	@echo Running go fmt on ALL_SRC ...
-	@$(GOFMT) -e -s -l $(ALL_SRC) > $(FMT_LOG)
-	./scripts/updateLicenses.sh >> $(FMT_LOG)
+lint:
+	golangci-lint -v run
+	./scripts/updateLicenses.sh > $(FMT_LOG)
 	./scripts/import-order-cleanup.sh stdout > $(IMPORT_LOG)
-	@[ ! -s "$(FMT_LOG)" -a ! -s "$(IMPORT_LOG)" ] || (echo "Go fmt, license check, or import ordering failures, run 'make fmt'" | cat - $(FMT_LOG) && false)
-
-.PHONY: go-lint
-go-lint:
-	@cat /dev/null > $(LINT_LOG)
-	@echo Running go lint...
-	@$(GOLINT) $(ALL_PKGS) | grep -v _nolint.go >> $(LINT_LOG) || true;
-	@[ ! -s "$(LINT_LOG)" ] || (echo "Lint Failures" | cat - $(LINT_LOG) && false)
+	@[ ! -s "$(FMT_LOG)" -a ! -s "$(IMPORT_LOG)" ] || (echo "License check or import ordering failures, run 'make fmt'" | cat - $(FMT_LOG) && false)
```

Comment:
This does not print import_log

Reply:
Address in [8b3a050](https://github.com/jaegertracing/jaeger/pull/3237/commits/8b3a050ff2123238276adb124fb7717f62e7c0ac)